hw_05/task_03.js

```diff
@@ -0,0 +1,27 @@
+// 1*. Создать программу, которая будет принимать на вход СЛОВО (создать переменную со словом), 
+//   и выводить в консоль количество гласных и согласных букв в этом слове. 
+//   Ответ должен выводиться шаблонным литералом вида word contains x vowels and y consonants
+
+
+const word = 'cream';
+const vowels  = ['a', 'e', 'i', 'o', 'u'];
+const consonants = ['b', 'c', 'd', 'f', 'g', 'h', 'j', 'k', 'l', 'm', 'n', 'p', 'q', 'r', 's', 't', 'v', 'w', 'y', 'z']
+
+let vowelsCounter = 0;
+let consonantsCounter = 0;
+const splitedWord = word.split('');
+
+for (let i=0; i < splitedWord.length; i++) {
+    if (vowels.includes[i]){
+        vowelsCounter++;}
+
```

Comment:
No check for consonants char, no increasing of consonants counter

Reply:
oops, lost a part of the code
fixed